bundles/org.openhab.binding.venstarthermostat/README.md

```diff
@@ -42,6 +42,12 @@ After adding the Inbox item, enter the user name and password from the physical
 | humidity           | Number             | Humidity                     |                                                        |
 
 
+### Thing Configuration 
```

Comment:
The "Thing Configuration" section should have a table (similar to the channels above), which lists and describes the different parameters. It should NOT include the textual configuration example (which usually is provided in a "Full Example" section). I see that the whole README does not really follow our template for the docs...

Reply:
Agreed, i'll clean up the README